routers/routes/routes.go

```diff
@@ -579,6 +579,14 @@ func RegisterRoutes(m *macaron.Macaron) {
 			m.Get("/milestones", repo.Milestones)
 		}, context.RepoRef())
 
+		m.Group("/branches", func() {
```

Comment:
I think this group needs `context.CheckUnit(models.UnitTypeCode)`

Reply:
done.